public/app/core/components/TimelineChart/TimelineChart.tsx

```diff
@@ -21,7 +21,7 @@ export interface TimelineProps extends Omit<GraphNGProps, 'prepConfig' | 'propsT
   tooltip?: VizTooltipOptions;
 }
 
-const propsToDiff = ['rowHeight', 'colWidth', 'showValue', 'mergeValues', 'alignValue', 'tooltip'];
+const propsToDiff = ['rowHeight', 'colWidth', 'showValue', 'mergeValues', 'alignValue', 'tooltip', 'height'];
```

Comment:
this will re-init the chart for each pixel that the chart is resized :(

Reply:
Maybe we can look for significant changes, defined as a function of row height.